From bd99949a4a54a502b209ed005b55561a86d04fd2 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@apache.org>
Date: Sat, 26 Jan 2013 20:33:41 +0000
Subject: [PATCH 1272/1357] MR1: MAPREDUCE-4962. jobdetails.jsp uses display name instead of real name to get counters. (sandyr via tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1@1438956 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 71a4429f006b489cda51862274751f446af7fbb6)

Reason: Bug
Ref: CDH-9004
Author: Sandy Ryza
---
 .../src/webapps/job/jobdetails.jsp                 |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/webapps/job/jobdetails.jsp b/hadoop-mapreduce1-project/src/webapps/job/jobdetails.jsp
index b89077f..e8e9ebd 100644
--- a/hadoop-mapreduce1-project/src/webapps/job/jobdetails.jsp
+++ b/hadoop-mapreduce1-project/src/webapps/job/jobdetails.jsp
@@ -378,7 +378,8 @@
       
       boolean isFirst = true;
       for (Counters.Counter counter : totalGroup) {
-        String name = counter.getDisplayName();
+        String displayName = counter.getDisplayName();
+        String name = counter.getName();
         String mapValue = decimal.format(mapGroup.getCounter(name));
         String reduceValue = decimal.format(reduceGroup.getCounter(name));
         String totalValue = decimal.format(counter.getCounter());
@@ -393,7 +394,7 @@
             <%
           }
           %>
-          <td><%=HtmlQuoting.quoteHtmlChars(name)%></td>
+          <td><%=HtmlQuoting.quoteHtmlChars(displayName)%></td>
           <td align="right"><%=mapValue%></td>
           <td align="right"><%=reduceValue%></td>
           <td align="right"><%=totalValue%></td>
-- 
1.7.0.4

